### PR TITLE
Make provision to provide kwargs to validation plug-ins.

### PR DESCRIPTION
**Pull request by jswelling**
Some new plugin validation tests will need access to the auth token, and the most efficient way to provide it is to allow Airflow to pass it down to the the plugins via kwargs.  This PR adds the necessary kwargs arguments to do so.

### CODE REVIEW

File: src/ingest_validation_tools/upload.py [hunk 1]
```diff
@@ -74,7 +75,7 @@ def get_errors(self):
         if reference_errors:
             errors['Reference Errors'] = reference_errors
 
-        plugin_errors = self._get_plugin_errors()
+        plugin_errors = self._get_plugin_errors(**kwargs)
```
Comment (mccalluc) [inline src/ingest_validation_tools/upload.py:78]:
```suggestion
        plugin_errors = self._get_plugin_errors(**plugin_kwargs)
```

**Review by mccalluc — CHANGES_REQUESTED** [review]
tweaks?

File: src/ingest_validation_tools/upload.py [hunk 2]
```diff
@@ -59,9 +59,10 @@ def __init__(self, directory_path=None, tsv_paths=[],
     #
     #####################
 
-    def get_errors(self):
+    def get_errors(self, **kwargs):
```
[the remark this answers is not in the document]
Comment (jswelling) [reply src/ingest_validation_tools/upload.py:62]: No objection, assuming you don't mind passing two instances of the auth token if you need it.

**Review by mccalluc — APPROVED** [review]
Merge at your convenience.

**Pull request merged**